client/session/profile_manager_test.go

```diff
@@ -314,6 +315,33 @@ func TestProfileManager(t *testing.T) {
 				Expect(err).To(BeNil())
 			})
 		})
+
+		g.Describe("GetAll", func() {
+			g.It("returns empty when no profiles exist", func() {
+				profiles, err := manager.GetAll()
+
+				Expect(err).To(BeNil())
+				Expect(profiles).To(BeEmpty())
+			})
+
+			g.It("returns all profiles", func() {
+				manager.Create(createValidProfile("profile-1"))
+				manager.Create(createValidProfile("profile-2"))
+				manager.Create(createValidProfile("profile-3"))
+
+				profiles, _ := manager.GetAll()
+
+				// Sort the slice to guarantee the order when comparing the results
+				sort.SliceStable(profiles, func(i int, j int) bool {
+					return profiles[i].Alias < profiles[j].Alias
+				})
+
+				Expect(len(profiles)).To(Equal(3))
+				Expect(profiles[0].Alias).To(Equal("profile-1"))
+				Expect(profiles[1].Alias).To(Equal("profile-2"))
+				Expect(profiles[2].Alias).To(Equal("profile-3"))
```

Comment:
It feels like there's probably a better way of doing this, but I couldn't spot anything obvious in the Gomega assertions. Any ideas / tips?

Reply:
Actually, I'm quite sure that's the way to go